minttea/event.ml

```diff
@@ -1,5 +1,8 @@
 open Riot
 
+type modifier = Ctrl
+type key_event = { key : string; modifier : modifier option }
```

Comment:
**suggestion**. Another thought is that to avoid the `(Some Ctrl)` wrapper when we want `Ctrl` we could make the `modifier` type just include a "no-modifier" variant like `None`:

```ocaml
type modifier = | None | Ctrl
```

Stuffing this type with combinations would also make it easier to check for Ctrl + Shift by just using `CtrlShift` instead of trying to have a set / list.

What do you think?

Reply:
I think this is really good, I just added option as it made sense that no modifiers could just be None, but either way works fine. But I do enjoy not have Some/Nones. To me this suggestion is great